build.gradle

```diff
@@ -0,0 +1,278 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This generated file contains a sample Java application project to get you started.
+ * For more details take a look at the 'Building Java & JVM projects' chapter in the Gradle
+ * User Manual available at https://docs.gradle.org/7.4.2/userguide/building_java_projects.html
```

Comment:
Not sure if all this blurb is really needed...

Reply:
Nuked locally, will show up in a future PR